### DISCUSSION TITLE
I can't login to my github account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#111161] [Other Feature Feedback, Questions, & Ideas]
I can't login to my github account as I had formatted my device and I can't login to my account due to 2 factor authentication and I can't login to ms authenticator also . Pls help .... Account name is gatik777

**Comment by lmdevv** [comment]
Do you have your backup codes? If not you can opt to use sms notifications or contact github support if available. It really depends on what you have opted in when you were in the account.

**Reply by ghost** [reply]
I had the backup codes. But unfortunately I deleted it. I am trying to recover it using re coverable software. And how to send sms message through github pls tell

**Reply by lmdevv** [reply]
SMS will only be available if you added a phone number when you had access to your account, if not it will not show. Unfortunately, it has happened to me before on ig, and Facebook customer service is the worst so I had to create an account, the only way is to contact Github Support and submit a request and wait.

**Reply by ghost** [reply]
I have created a support ticket and it's been 5 days they haven't responded to me

**Comment by devflash101** [comment]
Look carefully through your Gmail inbox.  You may find a mail from Github containing 2fa codes. With you good luck.

**Comment by ghost** [comment]
The problem is I can't. I have to be logged in

**Comment by ghost** [comment]
can u help a little bot more pls

**Comment by ghost** [comment]
can u help a little bit more pls

**Comment by samus-aran** [comment]
Hi @Gatik2007 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
